configs/Ec2UserdataUtils.psm1

```diff
@@ -409,3 +409,12 @@ function Disable-Firewall {
     Remove-Item -path $registryKey -recurse -force
   }
 }
+
+function Flush-EventLog {
+  <#
+  .Synopsis
+    Removes all entries from the event log. Used right before golden ami capture for a clean slate image.
+  #>
+  Write-Log -message 'flushing the Windows EventLog' -severity 'INFO'
+  wevtutil el | % { wevtutil cl $_ }
```

Comment:
I'm going to assume that this line is the same as the line above, only not in English :)